content/en/blog/_posts/2021-12-16-kubernetes-is-moving-on-from-dockershim.md

```diff
@@ -0,0 +1,96 @@
+components (like the kubelet) talk to Docker. It was built into Kubernetes code.
+Removing dockershim inevitably affects the entire CNCF family of projects beyond
+Kubernetes. Dependencies on Docker and dockershim have crept into various tools
+and projects in the Kubernetes ecosystem.
+
+There are a few container runtimes today that are often lighter, faster and more
+secure than when Docker was the only choice. By removing the dockershim CRI,
+we're embracing the first value of CNCF: "[Fast is better than slow](https://github.com/cncf/foundation/blob/master/charter.md#:~:text=Fast%20is%20better%20than%20slow)".
+Migrating to another runtime will ensure that applications are more portable,
+with fewer dependencies on the infrastructure. Your apps will be more secure,
+with new features unlocked. A different runtime will also let us develop
+features like cgroupv2 support faster.
+
+## Deprecation timeline
+
+We announced dockershim deprecation in December 2020.  Full removal is targeted
+to Kubernetes 1.24, which is planned for April 2022. This deprecation timeline
+aligns with our [deprecation policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-a-feature-or-behavior),
+which states that deprecated behaviors must function for no less than 1 year
+after their announced deprecation. We'll support version 1.23, which has
+dockershim, for another year in the Kubernetes project (and likely longer with
+managed Kubernetes providers). You'll have time to migrate. If you have more
+questions about the dockershim removal, refer to the [Dockershim Deprecation FAQ](https://k8s.io/dockershim).
+
+We asked you whether you feel prepared for the migration from dockershim in this
+survey: [Are you ready for Dockershim removal](https://kubernetes.io/blog/2021/11/12/are-you-ready-for-dockershim-removal/).
+We had about 600 responses. To everybody who took time filling out the survey,
+thank you.
+
+The results show that we still have a lot of ground to cover to help you to
+migrate smoothly. Several newer runtimes exist, and have been promoted
+extensively. However, a significant number of users still rely on dockershim,
+and sometimes have dependencies that need to be re-worked. Some of these
+dependencies are outside of your control. Based on the feedback received from
+you, here are some of the steps we are taking to help.
+
+## Our next steps
+
+Based on the feedback you provided, we'll be doing the following:
+
+- CNCF will sponsor work on updating documentation on how to migrate from
+  dockershim. We'll use survey feedback and our own experiences migrating from
+  dockershim to, try and cover as many use cases and known issues as possible.
+- We have begun, and will continue to, reach out to various projects and
+  vendors to make sure there is a path for customers migration.
+- The release team for Kubernetes 1.24 has set a high bar for documentation
+  readiness.
```

Comment:
```suggestion
- CNCF and the 1.24 release team are committed to delivering documentation in time for the 1.24 release. This includes a series of blog posts like this one, giving you timely advice on what upgrading to 1.24 entails, updating existing code samples, tutorials, and documentation, and producing a migration guide for concerned cluster operators. 
- We are reaching out to the rest of the CNCF community to help prepare and ready them for this change
- We are working with Mirantis and Docker to ensure that there is some path to comparability in the future.
```

cc @mrbobbytables and @amye - can we say that last bullet point?

Reply:
this may be a false promise that this will be path to compatibility. Some compatibility issues will be resolved, but not all.

Reply:
- CNCF and the 1.24 release team are committed to delivering documentation in time for the 1.24 release. This includes more informative blog posts like this one, updating existing code samples, tutorials, and tasks, and producing a migration guide for cluster operators. 
- We are reaching out to the rest of the CNCF community to help prepare them for this change.

Reply:
A suggest with @shannonxtreme's wording, which is 👍 

```suggestion
- CNCF and the 1.24 release team are committed to delivering documentation in time for the 1.24 release. This includes more informative blog posts like this one, updating existing code samples, tutorials, and tasks, and producing a migration guide for cluster operators.
- We are reaching out to the rest of the CNCF community to help prepare them for this change.
```